assets/darkutils/lang/zh_cn.json

```
@@ -1,127 +1,127 @@
 {
-    "__comment_file": "English Localization for Dark Utilities",
-    "__comment_creative_tabs": "Creative Tabs",
+    "__comment_file": "Dark Utilities的汉化翻译",
+    "__comment_creative_tabs": "创造模式面板名",
```

```suggestion
    "__comment_creative_tabs": "创造模式物品栏名",
```
（和上面一样，其实这玩意写啥都行，只是这样更正确）